src/pages/Landing/Landing.css

```diff
@@ -0,0 +1,41 @@
+.dark-mode {
+  background-color: rgb(34, 34, 34);
+  color: white;
+}
+.dark-mode .text-muted {
+  color: #b7b7b7;
```

Comment:
Try to follow one convention, either have your colors in `hex` code or in `rgb.`